app/uk/gov/hmrc/apiplatformmicroservice/apidefinition/models/APIDefinition.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiplatformmicroservice.apidefinition.models
+
+import uk.gov.hmrc.apiplatformmicroservice.apidefinition.models.APICategory.APICategory
+import uk.gov.hmrc.apiplatformmicroservice.apidefinition.models.APIStatus.APIStatus
+import uk.gov.hmrc.apiplatformmicroservice.apidefinition.models.HttpMethod.HttpMethod
+
+case class APIDefinition(serviceName: String,
+                         name: String,
+                         description: String,
+                         context: String,
+                         requiresTrust: Option[Boolean],
+                         isTestSupport: Option[Boolean] = None,
+                         versions: Seq[APIVersion],
+                         categories: Option[Seq[APICategory]] = None)
+
+object APICategory extends Enumeration {
+  type APICategory = Value
+
+  val EXAMPLE, AGENTS, BUSINESS_RATES, CHARITIES, CONSTRUCTION_INDUSTRY_SCHEME, CORPORATION_TAX, CUSTOMS, ESTATES, HELP_TO_SAVE, INCOME_TAX_MTD,
+  LIFETIME_ISA, MARRIAGE_ALLOWANCE, NATIONAL_INSURANCE, PAYE, PENSIONS, PRIVATE_GOVERNMENT,
+  RELIEF_AT_SOURCE, SELF_ASSESSMENT, STAMP_DUTY, TRUSTS, VAT, VAT_MTD, OTHER = Value
+}
+
+case class APIVersion(version: String,
+                      status: APIStatus,
+                      access: Option[APIAccess],
```

Comment:
Can we stop this being option and instead make it mandatory in the domain layer.  Only handle the option when loading from older data in mongo.

Reply:
I guess we could default it to public if it comes as none from mongo. We'll look into this.

Reply:
The biggest thing is not to splatter json formatters for the domain everywhere.  You will need a json formatter for this in connector.  Use the pattern of interface in services, implementation being the connector and the formatters being private to connector.